### PR TITLE
edk2 for turbot

### PR DESCRIPTION
**Pull request by BartekLew**

### CODE REVIEW

**Comment by pietrushnic** [comment]
@mscibisz please fix syntax.

File: _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Building EDK2 based firmware for MinnowBoard
+============================================
+
+There are some options to build firmware for MinnowBoard, a Bay Trail based
+SBC (Single Board Computer) from Intel. We prefer usually coreboot as simplest
+and fastest, open source solution, but sometimes we want to have UEFI
```
Comment (mscibisz) [inline _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md]: There are some options to build firmware for MinnowBoard, a Bay Trail-based SBC (Single Board Computer) from Intel. We usually prefer coreboot as the simplest and fastest open source solution, but sometimes we want to have UEFI interface.

File: _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+interface.
+
+UEFI itself doesn't cover whole boot procedure, so its open source reference
+implementation, EDK2 is not enough to build firmware for hardware plafrorm,
+we need to provide PI (Platform Initialization) phase implementation. In EDK2
+repository we can find only implementation for virtualization (OVMF), this
+option is covered in [this article](https://3mdeb.com/firmware/uefi-application-development-in-ovmf/#.WsOfOkuxVuE).
```
Comment (mscibisz) [inline _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md]: UEFI itself doesn't cover whole boot procedure, so its open source reference implementation, EDK2 is not enough to build firmware for hardware platforms...

File: _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+coreboot could be used to provide PI phase, but this procedure is mostly covered
+in [article on building coreboot for MinnowBoard](#Umiescic_link_tutaj), only we
+need to choose Tianocore payload. In this article we cover building UEFI
+firmware using binary objects from Intel. Whole procedure can be done using
+following script:
```
Comment (mscibisz) [inline _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md]: coreboot could be used to provide PI phase, but this procedure is mostly covered in the article on building coreboot for MinnowBoard, only we need to choose Tianocore payload. In this article, we cover building UEFI firmware using binary objects from Intel. Whole procedure can be done using following script:

File: _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    -v $PWD/MinnowBoard_MAX-0.97-Binary.Objects:/home/edk2/silicon \
+    -v $PWD/ccache:/home/edk2/.ccache \
+    3mdeb/edk2 /bin/bash -c 'cd edk2-platforms/Vlv2TbltDevicePkg/ &&
+        source Build_IFWI.sh MNW2 Debug'
+```
+
+in edk2-platforms repository we find open-source part of PI for various
+platforms including MinnowBoard. However we need also some closed code from
+Intel's site, which contains IP (Intelectual Property). Finaly we have to 
+fetch OpenSSL, which is another dependecy.
```
Comment (mscibisz) [inline _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md]: in edk2-platforms repository we find open-source part of PI for various platforms including MinnowBoard. However, we need also some closed code from Intel's site, which contains IP (Intelectual Property). Finally, we have to fetch OpenSSL, which is another dependency.

File: _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+When all those components are ready, we can build. We use dedicated docker
+images to avoid toolchain compatibility problems. So running docker we mount
+`edk2` (main repository), `edk2-platforms` and cache directory to respective
+mount points in the image (build script assume that they are all located in
+the same directory). So we enter `edk2-platforms/Vlv2TbltDevicePkg/` and run
+`source Build_IFWI.sh MNW2 Debug` (for DEBUG version). If build is successfully
```
Comment (mscibisz) [inline _posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md]: If the build is successfully...

**Comment by pietrushnic** [comment]
@BartekLew I think it would be economically more effective if you can use Grammarly by yourself starting with new blog posts. Let me know if you are fine with that.

Meanwhile please address all mentioned change requests.

**Comment by BartekLew** [comment]
@mscibisz @pietrushnic changes are done, github havent noticed because I renamed file to `post.md` by the way.

**Comment by pietrushnic** [comment]
@mscibisz if everything is fine please publish this blog post.

**Pull request merged**